test/pageobjects/main.page.ts

```diff
@@ -0,0 +1,232 @@
+import { $ } from "@wdio/globals";
+import Page from "./page.js";
+
+class MainPage extends Page {
+  public get logo() {
+    return $('div[data-testid="logo"]');
+  }
+  public get aboutUsLabel() {
+    return $('div[data-testid="content"]');
+  }
+  public get footerLogo() {
+    return $(
+      "//div[contains(concat(' ',normalize-space(@class),' '),'Footer_container')]/div[@data-testid='logo']"
+    );
+  }
+  public get privacyPolicyLink() {
+    return $('a[href="/privacy-policy/"]');
+  }
+  public get cookiePolicyLink() {
+    return $('a[href="/cookie-policy/"]');
+  }
+  public get termsConditionsLink() {
+    return $('a[href="/terms-conditions/"]');
+  }
+  public get forBuyersLabel() {
+    return $('div[class*="RentzilaForBuyers_title"]');
+  }
+  public get productsLink() {
+    return $('a[href="/products/"]');
+  }
+  public get tendersLink() {
+    return $('a[href="/tenders-map/"]');
+  }
+  public get requestsLink() {
+    return $('a[href="/requests-map/"]');
+  }
+  public get contactsLabel() {
+    return $('div[class*="RentzilaContacts_title"]');
+  }
+  public get emailLink() {
+    return $('a[href="mailto:info@rentzila.com.ua"]');
+  }
+  public get orderConsultationButton() {
+    return $("button=Замовити консультацію");
+  }
+  public get errorMessages() {
+    return $$('p[class*="ConsultationForm_error_message"]');
+  }
+  public get phoneNumberField() {
+    return $('input[id="mobile"]');
+  }
+  public get nameField() {
+    return $('input[name="name"]');
+  }
+  public get telegramCrossButton() {
+    return $('div[data-testid="crossButton"]');
+  }
+  public get proposesTitle() {
+    return $("h2=Послуги");
+  }
+  public get proposesElements() {
+    return $$(
+      "//section[@data-testid='services']//div[contains(concat(' ',normalize-space(@class),' '),'RentzilaProposes_name')]"
+    );
+  }
+  
```

Comment:
@Anastas11aZakharova Please, try not to leave the empty spaces, it causes readability issues, or add the comments to 'separate' the code.

Reply:
Fixed in the latest commit